### PR TITLE
SDK minor changes and command changes - closes #469, #471

### LINKED ISSUE #469: SDK minor changes/ command changes
After running through the procedure minor changes are required In  theNodejs section , Getting started page & Lisk SDK tutorials, 'Hello World'vetc.

### PR DESCRIPTION
**Pull request by chris529**
Closes #469, #471

### CODE REVIEW

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 1]
```diff
@@ -128,6 +128,11 @@ This will hold the logic to initialize and start the blockchain application.
 
 Configuration is necessary to provide the basic information for building the new application.
 
+Please execute the following command:
+....
+touch index.js
+....
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]: Please use this syntax to display code snippets. It enables syntax highlighting.

```suggestion
[source,bash]
----
touch index.js
----
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 2]
```diff
@@ -396,7 +401,10 @@ hello_world
 ....
 
 Add the new transaction type to your application, by registering it to the application instance inside of `index.js`.
-
+To create this file, please execute the command listed below:
+....
+touch index.js
+....
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]: Same here

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 3]
```diff
@@ -477,16 +485,16 @@ This will hold the client-side scripts.
...
-cd hello-world <1>
+cd hello_world <1>
 mkdir client <2>
 cd client <3>
 ----
 
-<1> Check that the root folder of the Hello-World application is open.
-<2> Create the folder for the client-side scripts inside the Hello-World folder.
+<1> Check that the root folder of the Hello_World application is open.
+<2> Create the folder for the client-side scripts inside the Hello_World folder.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]: When referring to the folder, use the code syntax. When referring to a name, start with a capital letter and leave spaces between words.

```suggestion
<1> Check that the root folder of the Hello World application is open.
<2> Create the folder for the client-side scripts inside the `hello_world` folder.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 4]
```diff
@@ -657,13 +666,13 @@ For further interaction with the network, it is possible to run the process in t
...
 ----
-cd hello-world <1>
+cd hello_world <1>
 pm2 start --name hello index.js <2>
 pm2 stop hello <3>
 pm2 start hello <4>
 ----
 
-<1> Navigate into the root folder of the Hello-World application.
+<1> Navigate into the root folder of the Hello_World application.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
<1> Navigate into the root folder of the Hello World application.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
LGTM!
Just left some small comments.

File: modules/ROOT/pages/tutorials/hello-world.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/tutorials/hello-world.adoc]: Please check https://asciidoctor.org/docs/user-manual/#applying-source-highlighting

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup.adoc [hunk 5]
```diff
@@ -130,3 +130,5 @@ To install the NPM package {url_npm_sdk}[lisk-sdk], please execute the following
 ----
 npm install lisk-sdk
 ----
+Install the `lisk-sdk` package as dependency of your project.
```
Comment (chris529) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
Install the `lisk-sdk` package as a dependency of your project.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 6]
```diff
@@ -566,7 +574,8 @@ To accomplish this, the API of the node is utilized and the created transaction
 
 As the API of every node is only accessible from the localhost by default, it is necessary to execute this query on the same server that your node is running on; unless the config was changed to <<step7,make your API accessible>> to others or to the public.
 
-IMPORTANT: Ensure your node is running, before sending the transaction.
+IMPORTANT: IMPORTANT: Ensure your node is running, before sending the transaction
```
Comment (chris529) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
IMPORTANT: Ensure your node is running  before sending the transaction.
```

**Review by chris529 — COMMENTED** [review]
Minor corrections

**Review by Tschakki — APPROVED** [review]

**Pull request merged**